packages/cmrjs/tests/cmr-utils/test-cmr-utils.js

```diff
@@ -1352,3 +1354,84 @@ test('buildCMRQuery works with if the input results list is empty', (t) => {
   const actual = buildCMRQuery(results);
   t.deepEqual(actual, expected);
 });
+
+test('updateECHO10Collection updates echo10 collection name and version', async (t) => {
+  const filename = 'tests/cmr-utils/data/meta.xml';
+  const cmrObject = await promisify(xml2js.parseString)(fs.readFileSync(filename, 'utf-8'), xmlParseOptions);
+  const updated = updateECHO10Collection(
+    cmrObject,
+    { name: 'a', version: 'b' }
+  );
+
+  t.is(updated.Granule.Collection.ShortName, 'a');
+  t.is(updated.Granule.Collection.VersionId, 'b');
+});
+
+test('updateCmrFileCollections updates Echo10Files at non-standard locations', (t) => {
+  const cmrObject = {
+    Granule: {
+      GranuleUR: 'MOD11A1.A2017200.h19v04.006.2017201090724',
+      InsertTime: '2017-11-20T23:02:40.055807',
+      LastUpdate: '2017-11-20T23:02:40.055814',
+      WhyThisAttribute: {
+        Collection: {
+          ShortName: 'MOD11A1',
+          VersionId: '006',
+        },
+      },
+    },
+  };
+  const updated = updateECHO10Collection(cmrObject, { name: 'a', version: 'b' });
+  t.is(updated.Granule.WhyThisAttribute.Collection.ShortName, 'a');
+  t.is(updated.Granule.WhyThisAttribute.Collection.VersionId, 'b');
+});
+
+test('updateCmrFileCollections updates Echo10Files when missing', (t) => {
+  const cmrObject = {};
+  const updated = updateECHO10Collection(cmrObject, { name: 'a', version: 'b' });
+  t.is(updated.Granule.Collection.ShortName, 'a');
+  t.is(updated.Granule.Collection.VersionId, 'b');
+});
+
+test('updateCmrFileCollections updates umm meta file', (t) => {
+  const filename = 'tests/cmr-utils/data/ummg-meta.json';
+  const cmrObject = JSON.parse(fs.readFileSync(filename, 'utf-8'));
+  const updated = updateUMMGCollection(cmrObject, { name: 'a', version: 'b' }, filename);
+
+  t.is(updated.CollectionReference.ShortName, 'a');
+  t.is(updated.CollectionReference.Version, 'b');
+});
+
+test('updateCmrFileCollections updates umm at non-standard locations', (t) => {
```

Comment:
What makes this a non-standard location?   Is it possible for it to be defined in both places?     Same question for UMM as well.

Reply:
I believe this is OBE via the huddle we had on 2/7 - this shouldn't be needed anymore.